tutorials/movies/the_cast_and_characters_they_played.md

```diff
@@ -135,43 +135,38 @@ namespace MovieTutorial.MovieDB.Entities
     //...
     public sealed class PersonRow : Row, IIdRow, INameRow
     {
-        //... remove QuickSearch from FirstName
-        [DisplayName("First Name"), Size(50), NotNull]
+        //... remove QuickSearch and NameProperty from FirstName
+        [DisplayName("Firstname"), Size(50), NotNull]
         public String Firstname
         {
-            get { return Fields.Firstname[this]; }
-            set { Fields.Firstname[this] = value; }
+            get => fields.Firstname[this];
```

Comment:
https://english.stackexchange.com/questions/78894/firstname-or-first-name
I think there should be a space in first name and last name (unlike surname) for display name attrs